tests/integration-tests/tests/dashboard/test_dashboard/test_dashboard/pcluster.config.yaml

```diff
@@ -1,7 +1,7 @@
 Image:
   Os: {{ os }}
 HeadNode:
-  InstanceType: {{ instance }}
+  InstanceType: t2.micro
```

Comment:
Is there a reason why we're changing the instance_type used for the `test_dashboard` integration tests?

Also, if we need to change it, I think the [test_dashboard.py::test_dashboard:](https://github.com/EddyMM/aws-parallelcluster/blob/de049facabdb086778c0a28a0f8f6403f4e905fd/tests/integration-tests/configs/common/common.yaml#L178) is the appropriate place to do so.

Reply:
I changed it so the tests can run faster, the test_dashboard integration tests didn't seem to have any tests that needed the default c5 large instance and they all worked perfectly fine with t2.micro instances. I will change it back for simplicity

Reply:
implemented